.github/workflows/nethermind-tests.yml

```diff
@@ -32,6 +31,23 @@ jobs:
     strategy:
       matrix:
         project:
+          - Ethereum.Abi.Test
+          - Ethereum.Basic.Test
+          - Ethereum.Blockchain.Block.Legacy.Test
+          - Ethereum.Blockchain.Block.Test
+          - Ethereum.Blockchain.Legacy.Test
+          - Ethereum.Blockchain.Pyspec.Test
+          - Ethereum.Blockchain.Test
+          - Ethereum.Difficulty.Test
+          - Ethereum.HexPrefix.Test
+          - Ethereum.KeyAddress.Test
+          - Ethereum.KeyStore.Test
+          - Ethereum.PoW.Test
+          - Ethereum.Rlp.Test
+          - Ethereum.Transaction.Test
+          - Ethereum.Transition.Test
+          - Ethereum.Trie.Test
+          - Ethereum.VM.Test
```

Comment:
Some are duplicated?

Reply:
They aren't. Do you see any? They are the same as before.